group.py

```diff
@@ -1,5 +1,83 @@
 """An example of how to represent a group of acquaintances in Python."""
 
 # Your code to go here...
+def max_age(group_information):
+    max_age = 0
+    people = None
+    for key, value in group_information.items():
+        if max_age < value['age']:
+            max_age = value['age']
+            people = key
```

Comment:
these all do work but can you do them in a comprehension expression. So for this function we could do (note: we're not asking for who has the maximum age, just what the maximum age is)
```suggestion
def max_age(group_information):
    ages = [person["age"] for person in group_information.values()]
    max_age = max(ages)
```